lib/Zef/CLI.rakumod

```diff
@@ -433,6 +433,8 @@ package Zef::CLI {
         # LOCAL PATHS
         abort "The following were recognized as file paths but don't exist as such - {@paths.grep(!*.IO.e)}"
             if +@paths.grep(!*.IO.e);
+        abort "The following were recognized as directory paths but don't contain a META6.json file - {@paths.grep(!*.IO.add('META6.json').e)}"
```

Comment:
Actually I think this needs to also check that it is a directory (and not an e.g. `.tar.gz` file) ala `@paths.grep(*.IO.d).grep(!*.IO.add('META6.json').e)`